django_react_proj/users-fe/src/components/RealTime/RealTime.js

```diff
@@ -1,18 +1,50 @@
 import React from 'react';
 import { format } from 'date-fns';
+import PropTypes from 'prop-types';
+import { toast } from 'react-toastify';
 
-const Time = () => {
+const Time = ({ users, firstNotifiaction, secondNotifiaction }) => {
   const [time, setTime] = React.useState(0);
   const currentCallback = () => {
     const date = new Date();
-    const dateformat = format(date, 'HH:mm');
-    setTime(dateformat);
+    setTime(date);
   };
   React.useEffect(() => {
     const handle = setInterval(currentCallback, 1000);
-    return () => clearInterval(handle);
+    return () => {
+      clearInterval(handle);
+    };
   }, []);
-  return time;
+  if (users) {
```

Comment:
This whole `if` block optionally can be a part of `currentCallback` where `time` is always get from `Date.now()` / `new Date()`. Then you don't need `time` state (`return format(...)` also can take `Date.now()` as it's the same value)